src/pages/AddBook.jsx

```diff
@@ -177,42 +178,54 @@ export default function AddBookPage() {
               </label>
             </form>
 
-            <div className="flex items-center h-16 bg-gray-50 ">
-              <button
-                className="h-10 ml-4 font-semibold text-white rounded-md bg-booklistBlue-dark w-28"
-                type="submit"
-                form="new book"
-                onClick={addBookHandler}
-              >
-                Save
-              </button>
+            <div className='flex items-center h-16 bg-gray-50 '>
+              {!title ? (
+                <h4>Choose a book to save changes.</h4>
+              ) : (
+                <button
+                  className='h-10 ml-4 font-semibold text-white rounded-md bg-booklistBlue-dark w-28'
+                  type='submit'
+                  form='new book'
+                  onClick={!listId ? setModalIsOpen : addBookHandler}
```

Comment:
This is not right, we want the modal to open if they navigate to the add book page before they create a list because there will be no lists for them to choose from. A book must be attached to a list